qemu/tests/cfg/virtio_fs_share_data.cfg

```diff
@@ -24,6 +22,14 @@
     size_mem1 = 4G
     use_mem_mem1 = no
     share_mem = yes
+    s390, s390x:
+        required_qemu = [5.2.0,)
+        take_regular_screendumps = no
```

Comment:
This may be an internal restriction?

Reply:
@fbq815 hello What do you think about `take_regular_screendumps`?
Is it added to the case or the loop?
I am not sure which loop to use for virtio_fs on S390 for testing.

Reply:
After discussion `take_regular_screendumps` will be added to the internal configuration